protected/humhub/modules/installer/views/config/cron.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+use humhub\modules\installer\forms\SampleDataForm;
+use humhub\modules\ui\form\widgets\ActiveForm;
+use yii\bootstrap\Html;
+
+?>
+<div id="cron" class="panel panel-default animated fadeIn">
+
+    <div class="panel-heading">
+        <?= Yii::t('InstallerModule.base', '<strong>Cron Jobs</strong>') ?>
+    </div>
+
+    <div class="panel-body">
+        <p><?= Yii::t('InstallerModule.base', 'Asynchronous jobs in HumHub are used to run potentially heavy or scheduled tasks such as sending out summary mails, notifications or search index optimization.'); ?></p>
+        <p>
+            <strong><?= Yii::t('InstallerModule.base', 'Cron job setup steps:') ?></strong>
+        </p>
+        <p>
+            <?= Yii::t('InstallerModule.base', 'Open the crontab of HumHub/PHP process user e.g. <code>www-data</code>.') ?>
+            <br>
+            <kbd>
+                crontab -e -u www-data
+            </kbd>
+        </p>
+
+        <p>
+            <?= Yii::t('InstallerModule.base', 'Add following line to the crontab:'); ?>
+            <br>
+            <kbd style="display: block; padding: 0.75rem 1rem;">
+                <span>
+                * * * * * /usr/bin/php /var/www/humhub/protected/yii queue/run >/dev/null 2>&1
```

Comment:
Personally just throwing an example such as this will confuse the end user, giving them an actual path for their specific webserver would be recommended, the same could be said for path to PHP as well.